### PR TITLE
Implemented possibility to configure the host.

### LINKED ISSUE #31: Configure the host to listen on
Awesome project! I also planned to implement something like reggie in order to manage our modules. So thanks for building this. It's great!

Do you think that it is a good idea to provide a start argument to configure the host. There are some cases where I want to run the reggie-server on my local machine and therefore don't want to expose it to the whole network.

Something like:

-s, --source Which kind of connections shall Reggie accept? [default: '0.0.0.0']

What do you think?

### PR DESCRIPTION
**Pull request by akoenig**
Closes #31

### CODE REVIEW

File: server.js [hunk 1]
```diff
@@ -19,13 +19,15 @@ var restify = require('restify')
 var argv = optimist
     .usage('Reggie wants to serve your packages!\nUsage: $0')
     //.demand(['d'])
-    .default({ d : path.join(process.cwd(), 'data'), p : 8080 })
+    .default({ d : path.join(process.cwd(), 'data'), p : 8080, s: '0.0.0.0' })
```
Comment (bajtos) [inline server.js]: This is line is getting too long in the sense of how much stuff is packed inside. Please extract the object with default values into a variable.

**Comment by bajtos** [comment]
Hello @akoenig, thank you for your pull request. The change looks generally good, there are two things to improve:
1. Please rename the option, AFAIK most applications use `--host` to configure the host:
   
   ```
   -h --host  Which host should Reggie listen on? [default: '0.0.0.0']
   ```
2. See the inline comment above.

**Comment by akoenig** [comment]
Sure, I will do that.

I also thought about "host" for this parameter. Unfortunately the "h" collides with the help command, so what do you think? No alias at all?

Edit: Ha, just saw your comment in line https://github.com/mbrevoort/node-reggie/blob/master/server.js#L18 :) So, yes, here you go :)

**Comment by bajtos** [comment]
>  I also thought about "host" for this parameter. Unfortunately the "h" collides with the help command, so what do you think? No alias at all?

IMO `--help` is the canonical way for getting help. It's quite common to use `-h` for something else, see e.g. GNU grep:

```
-h, --no-filename Suppress the prefixing of filenames on output when multiple files are searched.
```

I think it's ok to change `-h` from `--help` to `--host`.

File: README.md [hunk 2]
```diff
@@ -104,6 +104,7 @@ You can see the available options with `reggie-server -h`
 
 	-d, --data  Directory to store Reggie's data  [default: <cwd>/data]
 	-p, --port  Reggie's a good listener. What port should I listen on?  [default: 8080]
+	-s, --source  Which kind of connections shall Reggie accept? [default: '0.0.0.0']
```
Comment (bajtos) [inline README.md]: Please update the documentation to `-h, --host`.

**Comment by bajtos** [comment]
Almost good to merge - see the comment above.

BTW, sorry for reacting so late. Github unfortunately does not send a notification when a commit was added to a pull request. Therefore it's better to write an explicit comment, so that maintainers are notified about your update.

**Comment by akoenig** [comment]
Ah, sorry about that. Done :)

**Comment by bajtos** [comment]
Landed as 8f63bd0, released in v0.2.0. Thank you for your contribution.

**Pull request closed without merging**